app/main.py

```diff
@@ -1,3 +1,46 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: (int, float)) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: (int, float, Distance)) -> Distance:
+        if isinstance(other, Distance):
+            return Distance(km=self.km + other.km)
+        else:
+            return Distance(km=self.km + other)
+
+    def __iadd__(self, other: (int, float, Distance)) -> Distance:
+        if isinstance(other, Distance):
+            self.km += other.km
+        else:
+            self.km += other
+        return self
+
+    def __mul__(self, other: (int, float)) -> Distance:
+        return Distance(km=self.km * other)
+
+    def __truediv__(self, other: (int, float)) -> Distance:
+        return Distance(km=round((self.km / other), 2))
+
+    def __lt__(self, other: (int, float, Distance)) -> bool:
+        return self.km < other
+
+    def __gt__(self, other: (int, float, Distance)) -> bool:
+        return self.km > other
+
+    def __eq__(self, other: (int, float, Distance)) -> bool:
+        return self.km == other
+
+    def __le__(self, other: (int, float, Distance)) -> bool:
+        return self.km <= other
+
+    def __ge__(self, other: (int, float, Distance)) -> bool:
+        return self.km >= other
```

Comment:
Please add check as in __add__ method

Reply:
@katevasilenko do you mean to add _if isinstance(other, Distance)_